### PR TITLE
Update limits-ediscovery20.md

### PR DESCRIPTION
**Pull request by shelri**
These changes are a reflection to a larger clarification of what all counts towards the 4,000 character search limit customers are experiencing.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/limits-ediscovery20.md [hunk 1]
```diff
@@ -68,7 +68,8 @@ The limits described in this section are related to using the search tool on the
 |Maximum number of mailboxes or sites that can be searched in a single search. |No limit|
 |Maximum number of searches that can run at the same time. |No limit |
 |Maximum number of searches that a single user can start at the same time. |10 | 
-|Maximum number of characters for a search query (including operators and conditions). |10,000&nbsp;<sup>2</sup>|
+|Maximum number of characters for a search query (including operators and conditions). |10,000 &nbsp;<sup>2</sup>|
+|Maximum number of characters for a search query for Sharepoint and OneDrive For Business Sites (including operators and conditions). |10,000<br>4,000 with Wildcards&nbsp;<sup>2</sup>|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-ediscovery20.md]:
```suggestion
|Maximum number of characters for a search query for SharePoint and OneDrive for Business sites (including operators and conditions). |10,000<br>4,000 with Wildcards&nbsp;<sup>2</sup>|
```

File: microsoft-365/compliance/limits-ediscovery20.md [hunk 2]
```diff
@@ -127,7 +128,11 @@ Microsoft collects performance information for searches run by all organizations
 > [!NOTE]
 > <sup>1</sup> Any item that exceeds a single file limit will show up as a processing error.
 >
-> <sup>2</sup> When searching SharePoint and OneDrive for Business locations, the characters in the URLs of the sites being searched count against this limit.
+> <sup>2</sup> When searching SharePoint and OneDrive for Business locations, the characters in the URLs of the sites being searched count against this limit. The total number of characters consists of:<br>
+> - All Characters in both the Users and Filters fields
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-ediscovery20.md]:
```suggestion
> - All characters in both the Users and Filters fields.
```

File: microsoft-365/compliance/limits-ediscovery20.md [hunk 3]
```diff
@@ -127,7 +128,11 @@ Microsoft collects performance information for searches run by all organizations
...
+> - All Compliance Security Filters that apply to the user will be included in their search.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-ediscovery20.md]:
```suggestion
> - All compliance security filters that apply to the user will be included in their search.
```

File: microsoft-365/compliance/limits-ediscovery20.md [hunk 4]
```diff
@@ -127,7 +128,11 @@ Microsoft collects performance information for searches run by all organizations
...
+> - The characters from any location properties in the search (ExchangeLocation,PublicFolderLocation,SharpointLocation,ExchangeLocationExclusion,PublicFolderLocationExclusion,SharePointLocationExclusion, OneDriveLocationExclusion).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-ediscovery20.md]:
```suggestion
> - The characters from any location properties in the search (ExchangeLocation,PublicFolderLocation,SharPointLocation,ExchangeLocationExclusion,PublicFolderLocationExclusion,SharePointLocationExclusion, OneDriveLocationExclusion).
```

File: microsoft-365/compliance/limits-ediscovery20.md [hunk 5]
```diff
@@ -127,7 +128,11 @@ Microsoft collects performance information for searches run by all organizations
...
+> - Example: Including all SPO/ODB workloads will incur 6 characters as the word "ALL" will appear for both the SharePointLocation and OneDriveLocation
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-ediscovery20.md]:
```suggestion
> - Example: Including all SPO/ODB workloads will incur 6 characters as the word "ALL" will appear for both the SharePointLocation and OneDriveLocation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shelri please commit my suggestions.

**Comment by yogkumgit** [comment]
Gentle reminder @shelri Please update with feedback

**Comment by markjjo** [comment]
@yogkumgit I am working offline with @shelri and the PM on this PR. It's assigned to me and I will take ownership of any additional changes that need to be made.

**Comment by markjjo** [comment]
@shelri PR merged. These updates in the three pull requests that you submitted should be published by EOD tomorrow. Thanks again for submitting these.

**Pull request merged**